packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -99,10 +99,17 @@ function executeRequest({
         }
         const error: Object = new Error(parsed ? parsed.message : text)
         if (parsed) error.body = parsed
+
+        // Reset requestState to false
```

Comment:
Could we please make this comment a bit more descriptive? I.e., why are we resetting the request state? 

That we are setting it to false is already described well by the code already so currently it's a superfluous comment IMO.